bench.js

```diff
@@ -0,0 +1,33 @@
+var bench = require('fastbench')
+var memdb = require('memdb')
+var DAG = require('./')
+
+function buildAppend () {
+  var dag = DAG(memdb())
+  var next = 1
+
```

Comment:
this should use an appendStream (`dag.createAppendStream()`) since otherwise its gonna look up the heads on every append